hack/gen-docs.sh

```diff
@@ -21,88 +21,92 @@ set -o errexit
 PREVIOUS_DOCS_TOC="$(echo ${PREVIOUS_DOCS_VERSION} | tr . -)-toc"
 
-echo "Creating copy of site/_data/$PREVIOUS_DOCS_TOC.yml at site/_data/$NEW_DOCS_TOC.yml"
-cp site/_data/$PREVIOUS_DOCS_TOC.yml site/_data/$NEW_DOCS_TOC.yml
+echo "Creating copy of $DATA_DOCS_DIRECTORY/$PREVIOUS_DOCS_TOC.yml at $DATA_DOCS_DIRECTORY/$NEW_DOCS_TOC.yml"
+cp $DATA_DOCS_DIRECTORY/$PREVIOUS_DOCS_TOC.yml $DATA_DOCS_DIRECTORY/$NEW_DOCS_TOC.yml
 
-# 'git add' the previous version's ToC content as-is so we get a useful diff when we copy the main ToC in
+# 'git add' the previous version's ToC content as-is so we get a useful diff when we copy the master ToC in
 echo "Running 'git add' for previous version's ToC to use as a base for diff"
-git add site/_data/$NEW_DOCS_TOC.yml
+git add $DATA_DOCS_DIRECTORY/$NEW_DOCS_TOC.yml
 
-# now copy the main ToC so we can get a nice git diff of what changed since previous version
-echo "Copying site/_data/main-toc.yml to site/_data/$NEW_DOCS_TOC.yml"
-rm site/_data/$NEW_DOCS_TOC.yml && cp site/_data/main-toc.yml site/_data/$NEW_DOCS_TOC.yml
+# now copy the master ToC so we can get a nice git diff of what changed since previous version
+echo "Copying $DATA_DOCS_DIRECTORY/master-toc.yml to $DATA_DOCS_DIRECTORY/$NEW_DOCS_TOC.yml"
+rm $DATA_DOCS_DIRECTORY/$NEW_DOCS_TOC.yml && cp $DATA_DOCS_DIRECTORY/master-toc.yml $DATA_DOCS_DIRECTORY/$NEW_DOCS_TOC.yml
 
 # replace known version-specific links -- the sed syntax is slightly different in OS X and Linux,
 # so check which OS we're running on.
 if [[ $(uname) == "Darwin" ]]; then
     echo "[OS X] updating version-specific links"
-    find site/docs/${NEW_DOCS_VERSION} -type f -name "*.md" | xargs sed -i '' "s|https://velero.io/docs/main|https://velero.io/docs/$VELERO_VERSION|g"
-    find site/docs/${NEW_DOCS_VERSION} -type f -name "*.md" | xargs sed -i '' "s|https://github.com/vmware-tanzu/velero/blob/main|https://github.com/vmware-tanzu/velero/blob/$VELERO_VERSION|g"
+    find $DOCS_DIRECTORY/${NEW_DOCS_VERSION} -type f -name "*.md" | xargs sed -i '' "s|https://velero.io/docs/master|https://velero.io/docs/$VELERO_VERSION|g"
+    find $DOCS_DIRECTORY/${NEW_DOCS_VERSION} -type f -name "*.md" | xargs sed -i '' "s|https://github.com/vmware-tanzu/velero/blob/master|https://github.com/vmware-tanzu/velero/blob/$VELERO_VERSION|g"
 
-    echo "[OS X] Updating latest version in _config.yml"
-    sed -i '' "s/latest: ${PREVIOUS_DOCS_VERSION}/latest: ${NEW_DOCS_VERSION}/" site/_config.yml
+    echo "[OS X] Updating latest version in $CONFIG_FILE"
+    sed -i '' "s/latest: ${PREVIOUS_DOCS_VERSION}/latest: ${NEW_DOCS_VERSION}/" $CONFIG_FILE
 
     # newlines and lack of indentation are requirements for this sed syntax
     # which is doing an append
-    echo "[OS X] Adding latest version to versions list in _config.yml"
-    sed -i '' "/- main/a\\
-- ${NEW_DOCS_VERSION}
-" site/_config.yml
+    echo "[OS X] Adding latest version to versions list in $CONFIG_FILE"
+    sed -i '' "/- master/a\\
```

Comment:
This should be `main`

Reply:
There are a few more references to `master`, can you make sure they all point to `main`?

Reply:
Yes, thank you for all your feedbacks! I'll get this done this afternoon

Reply:
It should be fixed now